examples/adcp_example.ipynb

```diff
@@ -0,0 +1,1490 @@
+      "text/plain": [
+       "<Figure size 864x576 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Plot horizontal velocity direction\n",
+    "ax = plt.figure(figsize=(12,8)).add_axes([.14, .14, .8, .74])\n",
+    "plt.pcolormesh(t, dat_avg.range, dat_avg['U_dir'], cmap='twilight', shading='nearest')\n",
+    "# Plot the water surface\n",
+    "ax.plot(t, dat_avg.depth)\n",
+    "\n",
+    "# set up time on x-axis\n",
+    "ax.set_xlabel('Time')\n",
+    "ax.xaxis.set_major_formatter(dt.DateFormatter('%H:%M'))\n",
+    "\n",
+    "ax.set_ylabel('Altitude [m]')\n",
+    "ax.set_ylim([0, 12]);\n",
+    "plt.colorbar(label='Horizontal Vel Dir [deg CW from true N]');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Saving and Loading DOLfYN datasets\n",
+    "Datasets can be saved and reloaded using the `save` and `load` functions. Xarray is saved natively in netCDF format, hence the \".nc\" extension.\n",
+    "\n",
+    "Note: DOLfYN datasets cannot be saved using xarray's native `ds.to_netcdf`; however, DOLfYN datasets can be opened using `xarray.open_dataset`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Uncomment these lines to save and load to your current working directory\n",
+    "#dlfn.save(ds, 'your_data.nc')\n",
+    "#ds_saved = dlfn.load('your_data.nc')"
+   ]
+  }
+ ],
+ "metadata": {
+  "interpreter": {
+   "hash": "5cfd453a1a1cce2f32ea80f99ff7da863344217116d39185ac62b248c2577445"
+  },
+  "kernelspec": {
+   "display_name": "Python 3.8.12 64-bit ('base': conda)",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.7"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
This can be done pretty easily:
`dat = dlfn.load('sig1000_lander.nc')`
`avg_tool = dlfn.VelBinner(n_bin=dat.fs*300, fs=dat.fs)`
`adat = avg_tool.do_avg(dat)`
`adat = avg_tool.calc_tke(dat, out_ds=adat)`
`adat = avg_tool.calc_stress(dat, out_ds=adat)`
Except there's a lot of caveats with actually interpreting the results, so we haven't advertised it. We do have mostly-built source code for ADCP turbulence functions if you're interested (https://github.com/jmcvey3/dolfyn/blob/adcp-turbulence/dolfyn/adp/turbulence.py), but haven't moved forward on it for the same reasons.